components/gc-featured-link/gc-featured-link-fr.html

```diff
@@ -0,0 +1,63 @@
+---
+breadcrumb: [
+	{ "title": "GCWeb home", "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-fr.html" },
+	{ "title": "Fonctionalités provisoires", "link": "https://wet-boew.github.io/themes-dist/GCWeb/provisional-fr.html" }
```

Comment:
Remove the provisional breadcrumb, it is not a section anymore but a component state